index.html

```diff
@@ -227,6 +227,28 @@ <h2>LETS CONNECT</h2>
             </div>
         </section>
     </section>
+    <section class="contact-form">
+        <div class="wrapper-contact">
+            <h1 class="contact">Contact me</h1>
```

Comment:
- [ ] It is best SEO practice to use just one h1 tag for every HTML document. The h1 tag should be the main heading of your document. Kindly change every other h1 tag to another heading tag `h2 - h5`.